modules/lvms-using-raid-arrays.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-using-raid-arrays_{context}"]
+= Using software RAID arrays in {lvms}
+
+// Logical Volume Manager (LVM) does not support creating a software RAID.
+
+// You can create the RAID arrays using the `mdadm` utility. You can also use the RAID arrays that are created using the `mdadm` utility in LVM Storage. 
+
+Logical Volume Manager (LVM) does not support creating a software RAID. However, you can create the RAID arrays using the `mdadm` utility, and use the RAID arrays in the LVM Storage.
+
+.Prerequisites
+
+* You have created a software RAID during the {product-title} installation.
++
+[NOTE]
+====
+You can create the RAID arrays only during the installation of {product-title}. For information about creating RAID arrays in {product-title}, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/installing/installation-configuration#installation-special-config-raid_installing-customizing[Configuring a RAID-enabled data volume]
+====
+
+* You have installed LVM Storage.
+
+.Procedure
+
+. Open the `LVMCluster` custom resource (CR) YAML file.
++
+[NOTE]
+====
+If you have created the `LVMCluster`, then you can edit the existing `LVMCluster` CR. Else, you can perform this step while creating a new `LVMCluster`.
```

Comment:
Note that this only works for adding. One cannot replace the LVMCluster Devices

Reply:
Please revise this section as it is hard to edit an existing LVMCluster. We should recommend using a new CR for these setups (during install/setup time. not day2)